src/transaction/combine.ts

```diff
@@ -1,41 +1,63 @@
 import * as _ from 'lodash'
 import binary from 'ripple-binary-codec'
-import * as utils from './utils'
 import BigNumber from 'bignumber.js'
+import {ValidationError} from '../common/errors'
 import {decodeAccountID} from 'ripple-address-codec'
 import {validate} from '../common'
 import {computeBinaryTransactionHash} from '../common/hashes'
+import {JsonObject} from 'ripple-binary-codec/dist/types/serialized-type'
+
+/**
+ * The transactions should all be equal EXCEPT for the 'Signers' field. 
+ */
+ function validateAllSignedTransactionsAreEqual(transactions: Array<JsonObject>) {
```

Comment:
you've got some very long function names that could be terser without losing detail. 

this could be `validateSignedTransactionEquality` (probably shorter still but a start)

Reply:
actually - the `Signed` part is confusing, since we are validating transactions _regardless_ of signer. So it could just be `validateTransactionEquality`.

Reply:
Good point!